template/build/webpack.prod.conf.js

```diff
@@ -41,6 +42,22 @@ var webpackConfig = merge(baseWebpackConfig, {
     new ExtractTextPlugin({
       filename: utils.assetsPath('css/[name].[contenthash].css')
     }),
+	new PurifyCSSPlugin({
+        purifyOptions: {
+            whitelist: ['someclass']
```

Comment:
It is example how to do it  just in case. It change nothing if user have someclass or doesn't.